config/email.yaml.example

```diff
@@ -1,4 +0,0 @@
-# Outgoing email settings
```

Comment:
Why remove this? If you're deprecating it then it makes some sense to, but it's still fully supported from what I understand. (Do you mean to deprecate it?)